perfkitbenchmarker/benchmark_spec.py

```diff
@@ -115,6 +115,62 @@ def RedirectGlobalFlags(self):
     with self.config.RedirectFlags(FLAGS):
       yield
 
+  def ConstructVirtualMachineGroup(self, group_name, group_spec):
+    """
```

Comment:
To stay consistent, could you please make this docstring conform to this [styleguide](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments)?
If its a single line it should just be:
"""Construct the virtual machine(s) needed for a group."""